include/eve/arch/cpu/wide.hpp

```diff
@@ -401,9 +401,9 @@ namespace eve
     //! @brief Performs a compound bitwise and on all the wide lanes and assign the result to the current
     //! one
     template<value V>
-    friend EVE_FORCEINLINE auto operator&=(wide& w, V o) noexcept
+    friend EVE_FORCEINLINE bit_value_t<wide, V> operator&=(wide& w, V o) noexcept
```

Comment:
This feels weird.

1. &= should return reference. to *this. so wide&.
2. I'm not sure what's happening - are we allowing to bit and anything with anything?

what does `eve::wide<char> & double` do?